src/layouts/Default.tsx

```diff
@@ -42,23 +42,25 @@ export function Default (props: PropsWithChildren<ScanAppProps>): JSX.Element |
         <link rel='icon' type='image/png' sizes='48x48' href='/favicon.png' />
       </Head>
 
-      <NetworkProvider>
-        <WhaleProvider>
-          <StoreProvider state={props.initialReduxState}>
-            <StatsProvider>
-              <SupplyProvider>
-                <PoolPairsProvider>
-                  <Header />
-                  <main className='flex-grow'>
-                    {props.children}
-                  </main>
-                  <Footer />
-                </PoolPairsProvider>
-              </SupplyProvider>
-            </StatsProvider>
-          </StoreProvider>
-        </WhaleProvider>
-      </NetworkProvider>
+      <ThemeProvider theme='light'>
```

Comment:
Change this to whatever is on the local storage

Reply:
updated